packages/server/src/util/test-utils/seed.ts

```diff
@@ -146,53 +134,15 @@ export async function seedAll() {
 
 async function generateOrg(admin: User, numUsers: number, numRosterEntries: number, numUnits: number) {
   orgCount += 1;
-
-  const org = Org.create({
-    name: `Group ${orgCount}`,
-    description: `Group ${orgCount} for testing.`,
-    contact: admin,
-    indexPrefix: `testgroup${orgCount}`,
-    reportingGroup: `test${orgCount}`,
-    defaultMusterConfiguration: [],
-  });
+  const org = getOrgTestData(admin, orgCount);
   await org.save();
-
-  let reportSchemas = ReportSchema.create(defaultReportSchemas);
-  reportSchemas = reportSchemas.concat(ReportSchema.create(defaultReportSchemas));
-  for (let i = 0; i < reportSchemas.length; i++) {
-    const report = reportSchemas[i];
-    report.org = org;
-    if (i > 0) {
-      report.id += i.toString();
-    }
-  }
+  const reportSchemas = getReportSchemaTestData(org);
   await ReportSchema.save(reportSchemas);
-
-  const customColumn = CustomRosterColumn.create({
-    org,
-    display: `My Custom Column ${orgCount}`,
-    type: RosterColumnType.String,
-    phi: false,
-    pii: false,
-    required: false,
-  });
+  const customColumn = getCustomRosterColumnTestData(org, orgCount);
   await customColumn.save();
 
   // Add Units
-  const units: Unit[] = [];
-  for (let i = 1; i <= numUnits; i++) {
-    unitCount += 1;
-    const unit = Unit.create({
-      org,
-      name: `Unit ${unitCount}`,
-      musterConfiguration: [
-        { days: 62, startTime: '00:00', timezone: 'America/Los_Angeles', durationMinutes: 120, reportId: reportSchemas[0].id },
-        { startTime: '2020-01-02T02:00:00.000', timezone: 'America/Los_Angeles', durationMinutes: 120, reportId: reportSchemas[1].id },
-      ],
-      includeDefaultConfig: true,
-    });
-    units.push(unit);
-  }
+  const units = getUnitsTestData(numUnits, org, reportSchemas);
   await Unit.save(units);
 
   const groupAdminRole = createGroupAdminRole(org);
```

Comment:
Should we have a corresponding generator for this (the role) also?

Reply:
Yes, and there is lot more that should be extracted. I concentrated on what is needed to get roster and all the related data generated. 
I think this work needs to be part of a new task so that I can move back to completing my primary assignment. 
This is a good discussion and some initial work done with more needed.